src/components/popup/Route.tsx

```diff
@@ -14,32 +14,18 @@ const Route: typeof BaseRoute = ({ path, component, children }) => {
     ? children(params)
     : children;
 
-  return (
-    <AnimatePresence initial={false}>
-      {matches && <Page>{routeContent}</Page>}
-    </AnimatePresence>
-  );
+  return matches ? <Page>{routeContent}</Page> : null;
 };
 
-export const Wrapper = styled(motion.div)<{
-  responsive?: boolean;
-  expanded?: boolean;
-}>`
+const PageWrapper = styled(motion.main)`
   position: relative;
-  width: ${(props) => (props.responsive ? "100%" : "377.5px")};
-  min-height: ${(props) => (props.expanded ? "100vh" : "600px")};
-  max-height: max-content;
-  background-color: rgb(${(props) => props.theme.background});
-`;
-
-const PageWrapper = styled(Wrapper)`
```

Comment:
There were some unnecessary containers (HTML) that is now gone, so the HTML structure should look much simpler:

    body > #__plasmo > main#Page